<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>626</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Seleção da base</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>600</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>resources/WhatsApp Image 2024-12-06 at 15.05.47.jpeg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="styleSheet">
             <string notr="true">QGroupBox {
    border: 3px solid #e8e8e8; /* Borda de 3px com cor #e8e8e8 */
    border-radius: 10px;       /* Cantos arredondados (10px de raio) */
    background-color: white;   /* Fundo branco */
}</string>
            </property>
            <property name="title">
             <string/>
            </property>
            <widget class="QPushButton" name="btnSelectBase">
             <property name="geometry">
              <rect>
               <x>160</x>
               <y>470</y>
               <width>131</width>
               <height>41</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton:enabled {
	font: 12pt &quot;Bernard MT Condensed&quot;;
	border: none;
	background-color: #133b70;
	color: white;
	border-radius: 13;
}
QPushButton:disabled {
	font: 12pt &quot;Bernard MT Condensed&quot;;
	border: none;
	background-color: Gainsboro;
	color: DimGray;
	border-radius: 13;
}
</string>
             </property>
             <property name="text">
              <string>Selecionar</string>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_2">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>470</y>
               <width>131</width>
               <height>41</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	font: 12pt &quot;Bernard MT Condensed&quot;;
	border: none;
	background-color: #e8e8e8;
	color: black;
	border-radius: 13;
}
</string>
             </property>
             <property name="text">
              <string>Cancelar</string>
             </property>
            </widget>
            <widget class="QWidget" name="verticalLayoutWidget_5">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>291</width>
               <height>411</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <property name="spacing">
               <number>8</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <property name="spacing">
                 <number>2</number>
                </property>
                <item>
                 <widget class="QLabel" name="OrbitLabel_4">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>5000</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Bernard MT Condensed</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">
	font: 12pt &quot;Bernard MT Condensed&quot;;
</string>
                  </property>
                  <property name="text">
                   <string>Lado</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cbSide">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QComboBox{
	font: 10pt &quot;Bernard MT Condensed&quot;;
}
</string>
                  </property>
                  <property name="currentText">
                   <string>Escolha uma opção</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>Escolha uma opção</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Direito</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Esquerdo</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="OrbitLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>5000</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Bernard MT Condensed</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">
	font: 12pt &quot;Bernard MT Condensed&quot;;
</string>
                  </property>
                  <property name="text">
                   <string>Numeração</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cbSize">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QComboBox{
	font: 10pt &quot;Bernard MT Condensed&quot;;
}
</string>
                  </property>
                  <property name="currentText">
                   <string>Escolha uma opção</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>Escolha uma opção</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>19</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>20</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>21</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>22</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>23</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>24</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>25</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>26</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>27</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>28</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>29</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>30</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>31</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>32</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>33</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>34</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>35</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>36</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>37</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>38</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>39</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>40</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>41</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>42</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>43</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>44</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>45</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QLabel" name="OrbitLabel_2">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>5000</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Bernard MT Condensed</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">
	font: 12pt &quot;Bernard MT Condensed&quot;;
</string>
                  </property>
                  <property name="text">
                   <string>Espessura da palmilha (E)</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cbThickness">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QComboBox{
	font: 10pt &quot;Bernard MT Condensed&quot;;
}
</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>Escolha uma opção</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>3</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>4</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>5</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>6</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>7</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>8</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>9</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>10</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QLabel" name="OrbitLabel_3">
                  <property name="minimumSize">
                   <size>
                    <width>120</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>5000</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Bernard MT Condensed</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <italic>false</italic>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">
	font: 12pt &quot;Bernard MT Condensed&quot;;
</string>
                  </property>
                  <property name="text">
                   <string>Altura do calcanhar (D)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cbHeel">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QComboBox{
	font: 10pt &quot;Bernard MT Condensed&quot;;
}
</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>Escolha uma opção</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>3</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>4</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>5</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>6</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>7</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>8</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>9</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>10</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
